suite/runsuite.cmake

```diff
@@ -546,6 +550,30 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
+if (ARCH_IS_X86 AND UNIX AND cross_riscvxx_linux_only)
```

Comment:
We may want this to behave in the same way as the aarchxx and android cross tests: to be enabled in local suite runs too and not only triggered by this GA CI variable.  The other cross tests will operate locally if the cross compiler is built, so you can run the whole suite on your local machine and include the cross tests.

Reply:
Done.